python_translators/translators/google_translator.py

```diff
@@ -35,7 +36,11 @@ def __init__(self, source_language: str, target_language: str, key: str, transla
         )
 
         self.key = key
-        self.translation_service = build('translate', 'v2', developerKey=key)
+        try:
+            self.translation_service = build('translate', 'v2', developerKey=key)
+        except HttpError as e:
+            responseString = f'error: {e} rrrrrrrrrrrresponse: {e.resp}  ssssssssssstatus: {e.resp.status}'
```

Comment:
is this a pirate log? :)

Reply:
hahah :smile: 
A no one was supposed to see that..